scripts/run_tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Make sure we're running from the bleach repository directory and
+# not this directory.
+THISDIR=$(basename `pwd`)
+if [[ "${THISDIR}" == "scripts" ]]; then
+    cd ..
+fi
```

Comment:
Tox runs commands via subprocess, so it can't run a command in a different directory without switching to that directory first (or us adding a bunch of shenanigans). So then we need this script to run both from the scripts directory as well as the bleach repository root. This bit figures out what the working directory is and fixes it if it's the wrong one.